tests/internal/CMakeLists.txt

```diff
@@ -115,7 +115,21 @@ if(ENABLE_GTEST)
     blt_add_test(NAME t_header_only_smoke
                  COMMAND t_header_only_smoke)
 
-    if ( NOT CMAKE_HOST_WIN32 )
+    ####################################################
+    # Tests blt_combine_static_libs macro
+    # Note: Does not currently work on Windows or BGQ
+    ####################################################
+    set(_try_combine_static_libs ON)
+
+    if(CMAKE_HOST_WIN32)
+        set(_try_combine_static_libs OFF)
+    endif()
+
+    if ("$ENV{SYS_TYPE}" STREQUAL "bgqos_0")
```

Comment:
I had to disable the ``combine_static_libs`` test on ``bgq`` due to the way that static/shared libraries interact there.  I can create a ticket if anyone thinks this is worth investigating.